maxim/tests/test_decorators.py

```
@@ -79,9 +80,11 @@ def test_span():
 
         @trace(logger=self.logger)
         def testing(test):
-            print("inside testing")
-            current_trace().set_input("test_input")
-            current_trace().set_output("test_output")
+            trace = current_trace()
+            if trace is None:
+                raise ValueError("current_trace is None")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)